lib/h3.rb

```diff
@@ -36,16 +40,18 @@ module H3
                   :h3UnidirectionalEdgeIsValid,
                   [ H3_INDEX ],
                   :bool
-  attach_function :origin_from_unidirectional_edge,
-                  :getOriginH3IndexFromUnidirectionalEdge,
-                  [ H3_INDEX ],
-                  H3_INDEX
-  attach_function :h3_valid, :h3IsValid, [ H3_INDEX ], :bool
+  attach_function :hexRange, [ H3_INDEX, :int, :pointer ], :int
+  attach_function :hexRing, [H3_INDEX, :int, :pointer], :void
   attach_function :hex_area_km2, :hexAreaKm2, [ :int ], :double
   attach_function :hex_area_m2, :hexAreaM2, [ :int ], :double
+  attach_function :kRing, [H3_INDEX, :int, :pointer], :void
   attach_function :max_h3_to_children_size, :maxH3ToChildrenSize, [ H3_INDEX, :int ], :int
   attach_function :max_kring_size, :maxKringSize, [ :int ], :int
   attach_function :num_hexagons, :numHexagons, [ :int ], H3_INDEX
+  attach_function :origin_from_unidirectional_edge,
```

Comment:
I think you forgot to update this line.

Reply:
Actually we now have `origin_from_unidirectional_edge`, `destination_from_unidirectional_edge`, and `origin_and_destination_from_unidirectional_edge` which returns both.

Reply:
Got it.